src/components/accommodation/form/sections/AccommodationFormAdditional.jsx

```diff
@@ -18,7 +18,7 @@ const AccommodationFormAdditional = () => {
             </AccommodationFormSectionHeader>
             <Row>
                 <Col xs={12}>
-                    <AccommodationFormDescription />
+                    {/*<AccommodationFormDescription />*/}
```

Comment:
This was actually discussed and will appear on the API side, even if it's not there yet.